jasc/BUILD

```diff
@@ -0,0 +1,223 @@
+# Schedules for JAX.
+
+load("@rules_python//python:defs.bzl", "py_library")
+load("@pybind11_bazel//:build_defs.bzl", "pybind_extension")
+load("@llvm-project//mlir:build_defs.bzl", "cc_headers_only")
+
+package(
+    # default_applicable_licenses = ["//third_party/mlir_edge:license"],
+    default_visibility = ["//visibility:public"],
+)
+
+py_library(
+    name = "jasc",
+    srcs = ["jasc.py", "__init__.py"],
```

Comment:
Nit: can we run bulidifier from bazeltools to reformat this?  I suspect each file should be on a separate line.